src/assets/translations/szit.json

```diff
@@ -16251,7 +16251,7 @@
                   "verse": "22"
                 },
                 {
-                  "text": "Jézus így felelt: „Ha valamit tehetsz?… Minden lehetséges annak, aki hisz.”",
+                  "text": "Jézus így felelt: „Ha valamit tehetsz…? Minden lehetséges annak, aki hisz.”",
```

Comment:
This is critical, szentiras.hu API error:
https://szentiras.hu/api/idezet/Mk9,23/SZIT

I checked the text bible, and there "Ha valamit tehetsz…?" presented in mk9,23.